### PR TITLE
Corrects logging set in function apps

### PR DESCRIPTION
**Pull request by jrabbott**
Corrects logging to use app insights connection string (key is becoming redundant).

Adds structured logging using Serilog.

### CODE REVIEW

File: platform/terraform/main.tf
Comment (WolfyUK) [inline platform/terraform/main.tf:1]: Do we run `terraform fmt` in the pipeline? The spacing change here might break it if so.

Comment (WolfyUK) [reply platform/terraform/main.tf:1]: (Also in the other `.tf` changes)

Comment (jrabbott) [reply platform/terraform/main.tf:1]: The spacing change won't break it. However I had considered adding to the pipeline, but a number of reason why I didn't - help to have a chat.

Comment (jrabbott) [reply platform/terraform/main.tf:1]: Will run fmt locally and push.... reads nicer

**Review by WolfyUK — APPROVED** [review]

**Pull request merged**